.github/workflows/e2e_tests.yml

```diff
@@ -13,9 +13,16 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: recursive
+      - uses: shivammathur/setup-php@v2
```

Comment:
The plugin now has composer dependencies for releasing - we need them when executing the tests